inspire_json_merger/config.py

```diff
@@ -99,6 +99,18 @@ class ArxivOnArxivOperations(MergerConfigurationOperations):
         'persistent_identifiers': D.FALLBACK_KEEP_HEAD,
         'preprint_date': D.FALLBACK_KEEP_HEAD,
         'publication_info': D.FALLBACK_KEEP_HEAD,
+        # Fields bellow are merged and conflicts are ignored, so for those fields we stay with previous merge behaviour.
+        'abstracts': D.FALLBACK_KEEP_UPDATE,
```

Comment:
what about references? DO we have to specify an operation?

Reply:
What about references? They are not specified in `conflict_filters` so they will be processed like any other key.